canaries/pcengines/canary-001-2019.txt

```diff
@@ -88,4 +88,4 @@ canary in the 3mdeb-secpack.git repo, and (2) via digital signatures
 on the corresponding 3mdeb-secpack.git repo tags. [2]
 
 [2] Don't just trust the contents of this file blindly! Verify the
-digital signatures!
\ No newline at end of file
+digital signatures!
```

Comment:
@macpijan I really doubt we should touch canaries even if those contain bugs although I can't see any mistake in this line with my bare eyes.

Reply:
@pietrushnic Dropped, and excluced canaries.
The change was auto fix from pre-commit. We are missing EOL in canary template and it was propagated to all of them.

Reply:
@macpijan I understand that canaries have typos and formatting issues and understand that pre-commit did fixes automatically, but we signed a broken form. Let's treat it as an additional layer of protection introduced not intentionally.

There are still typo fixes in PC Engines canaries.

Reply:
@macpijan ping?

Reply:
@pietrushnic canaries are no longer modified